tests/Tests/Serialise.hs

```diff
@@ -175,6 +177,7 @@ testTree = testGroup "Serialise class"
       , mkTest (T :: T [Int])
       , mkTest (T :: T UTCTime)
       , mkTest (T :: T Version)
+      , mkTest (T :: T Fingerprint)
```

Comment:
This line seems to be failing to build in travis in one config. Could you have a look?